docs/agents/agent-development.md

```diff
@@ -195,6 +195,24 @@ Service metadata relates to the service/application being monitored:
 
 For official Elastic agents, the agent name should just be the name of the language for which the agent is written, in lower case.
 
+### Cloud Provider Metadata
+
+Cloud provider metadata is collected from local cloud provider metadata services:
+
+- availability_zone
```

Comment:
Maybe link to https://github.com/elastic/apm-server/blob/master/docs/spec/cloud.json vs repeating here?

Reply:
We repeat the list for all of the other metadata types. Maybe I'll add the link and also leave the list?